src/ssl.c

```diff
@@ -18028,6 +18020,14 @@ size_t wolfSSL_get_client_random(const WOLFSSL* ssl, unsigned char* out,
 
 #endif /* OPENSSL_EXTRA */
 
+    void wolfSSL_ERR_clear_error(void)
```

Comment:
Do we really need the function wolfSSL_ERR_clear_error to be available in all cases? Perhaps just wrap in #if defined(OPENSSL_EXTRA) || defined(DEBUG_WOLFSSL_VERBOSE)?